com.unity.netcode.gameobjects/Tests/Runtime/NetworkVariableTests.cs

```diff
@@ -3,9 +3,101 @@
 using UnityEngine;
 using UnityEngine.TestTools;
 using NUnit.Framework;
+using Unity.Collections;
+
 
 namespace Unity.Netcode.RuntimeTests
 {
+    public unsafe struct FixedStringStruct : INetworkSerializable
+    {
+        private const int k_MaxSize = 128;
+        public fixed char FixedString[k_MaxSize];
+        public int StringSize;
+
+        public void SetIt(string valueToSet)
+        {
+            Assert.False(valueToSet.Length >= k_MaxSize);
+            fixed(char* charPtr = FixedString)
+            {
+                for(int i = 0; i < valueToSet.Length; i++)
+                {
+                    charPtr[i] = valueToSet[i];
+                }
+            }
+            StringSize = valueToSet.Length;
+        }
+
+        public string GetIt()
+        {
+            var retValue = string.Empty;
+            fixed (char* charPtr = FixedString)
+            {
+                for (int i = 0; i < StringSize; i++)
+                {
+                    retValue += FixedString[i];
+                }
+            }
+            return retValue;
+        }
+
+        public void NetworkSerialize(NetworkSerializer serializer)
+        {
+            if (serializer.IsReading)
+            {
+                serializer.Serialize(ref StringSize);
+
+                var charArray = new char[StringSize];
+                serializer.Serialize(ref charArray);
+
+                fixed (char* charPtr = FixedString)
+                {
+                    for (int i = 0; i < StringSize; i++)
+                    {
+                        charPtr[i] = charArray[i];
+                    }
+                }
+            }
+            else
+            {
+                serializer.Serialize(ref StringSize);
+
+                var charArray = new char[StringSize];
+                fixed (char* charPtr = FixedString)
+                {
+                    for (int i = 0; i < StringSize; i++)
+                    {
+                        charArray[i] = charPtr[i];
+                    }
+                }
+                serializer.Serialize(ref charArray);
+            }
+        }
+    }
+
+    public struct FixedString32Struct : INetworkSerializable
+    {
+        public FixedString32 String;
+        public void NetworkSerialize(NetworkSerializer serializer)
+        {
+            if (serializer.IsReading)
```

Comment:
Likely could make this even better when @ShaduaxCat stuff lands because then we can just write the length of the FixedString and then also write the raw byte* and then do the same for the read so there is no need to create the managed string.

Reply:
> > > So we have a FixedString class in Collections... its tested, it works, we use it for UTP today.
> > > https://docs.unity3d.com/Packages/com.unity.collections@1.0/api/Unity.Collections.FixedString.html
> > 
> > 
> > @andrews-unity
> > So what is the trick to get Unity.Collections working in our runtime test assembly definition? :)
> > We want to nix the other unsafe fixed string struct and run with FixedString32...and what not.
> 
> You should just be able to reference collections in the test assembly. I pulled down this branch and it seemed fine, and I was able to run the tests.

The reason this PR is failing is because that assembly can't find the Unity.Collections namespace.  It is failing on a compilation error.

Reply:
Tests are green now :D the issue was with the fact that collections is not a direct dependency of the NGO package and so when the upm-ci does it isolation testing it won't find them. I added collections as a dependency

Reply:
OMG... Ok... so this package 
`"com.unity.collections": "0.12.0-preview.13"`
Wasn't in the packages... haha.  Ok, thanks for helping for sure!  (lol)
Will remember to check that next time this happens.